sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/protocol/share_rest_client.hpp

```diff
@@ -330,14 +231,14 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
       std::string Prefix;
       std::string PreviousContinuationToken;
       int32_t PageSizeHint = int32_t();
-      Models::FilesAndDirectoriesListSinglePage SinglePage;
+      FilesAndDirectoriesListSinglePage SinglePage;
```

Comment:
Move the two members of `FilesAndDirectoriesListSinglePage` here?

Reply:
It has already been flattened in `ListFilesAndDirectoriesSinglePageResult`. This one is not visible to customers.

Reply:
But it's not in `Details` namespace, is it?

Reply:
Yeah, you are right. Will push a commit to move this kind of response/result into details namespace.